unittests/util/TestGlobals.h

```diff
@@ -25,6 +25,13 @@
  * Contains global variables for use in tests.
  */
 struct TestGlobals {
+    TestGlobals(TestGlobals const&) = delete;
```

Comment:
This is not a great change imo: all CTOR and operators were together before, now they are not

Reply:
Ah i see what happened there. You may want to merge latest develop - i already have fixed this yesterday 👍